include/experimental/__p2630_bits/submdspan_mapping.hpp

```diff
@@ -84,14 +85,27 @@ any_slice_out_of_bounds(const extents<IndexType, Exts...> &exts,
 }
 
 // constructs sub strides
+template<class T, size_t N>
+struct sub_strides
+{
+  T values[N];
```

Comment:
```suggestion
  T values[N > 0 ? N : 1];
```

Reply:
Done (and delete the specialization below) unless you prefer the specialization path.